### PR TITLE
Add Simplified Chinese translation

### PR DESCRIPTION
**Pull request by dujiajun**

### CODE REVIEW

**Comment by merefield** [comment]
@jumagura could you please take a look as per conversation on Discord?

**Comment by jumagura** [comment]
@dujiajun Thanks a lot for this contribution. 👏🏽 👏🏽  My apologize for the delay reviewing this PR. Thanks @merefield for the notification

**Review by jumagura — APPROVED** [review]

File: config/locales/client.zh_CN.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+en:
```
Comment (jumagura) [inline config/locales/client.zh_CN.yml:1]: change 'en' to 'zh'
Since that's the code for simplified Chinese

**Review by jumagura — CHANGES_REQUESTED** [review]
Just a small change on the first line
change 'en' to 'zh'
Since that's the code for simplified Chinese

File: config/locales/client.zh_CN.yml (hunk 1 above)
Comment (dujiajun) [reply config/locales/client.zh_CN.yml]: I see. I removed the default placeholder locale for `zh` and update `zh_CN` translation.

**Pull request merged**